.github/workflows/playwright.yaml

```
@@ -89,24 +88,37 @@ jobs:
         shard: [1, 2, 3]
 
     steps:
-      - name: Checkout 📥
+      - name: Checkout
         uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
 
       - name: Set backend branch
         id: backend-branch
         run: echo "branch=develop" >> $GITHUB_OUTPUT
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:1:26: Double quote to prevent globbing and word splitting

(shellcheck)